srcpkgs/ardour/patches/fix-musl-platform-detection.patch

```diff
@@ -0,0 +1,13 @@
+On musl libc only __linux__ is defined
+
+--- libs/vst3/pluginterfaces/base/fplatform.h
++++ libs/vst3/pluginterfaces/base/fplatform.h
```

Comment:
Please look into upstreaming this, if you can :)

Reply:
just did ^^ https://github.com/Ardour/ardour/pull/596